### PR TITLE
[Core] Add CreateQuadraturePoints to python

### PR DESCRIPTION
**Pull request by tteschemacher**
Hi,

this PR adds the required interface for the quadrature points to python, including the GeometriesVector.

### CODE REVIEW

File: kratos/python/containers_interface.h [hunk 1]
```diff
@@ -97,6 +97,26 @@ class PointerVectorPythonInterface
...
+
+    void CreateInterface(pybind11::module& m, std::string ContainerName)
+    {
+        py::class_<TContainerType, typename TContainerType::Pointer  >(m, ContainerName.c_str())
+            .def(py::init<>())
+            .def("__len__", [](TContainerType& self) {return self.size(); })
+            .def("__setitem__", [](TContainerType& self, unsigned int i, typename TContainerType::value_type& value) {self[i] = value; })
+            .def("__getitem__", [](TContainerType& self, unsigned int i) {return self(i); })
+            .def("__iter__", [](TContainerType& self) {return py::make_iterator(self.begin(), self.end()); }, py::keep_alive<0, 1>())
+            ;
```
Comment (pooyan-dadvand) [inline kratos/python/containers_interface.h]: You may add the append with pointer:
```c++
.def("append",    [](TContainerType& self, typename TContainerType::value_type::Pointer pValue){self.push_back(pValue);}  )
```

**Review by pooyan-dadvand — APPROVED** [review]
👍

**Comment by philbucher** [comment]
still need this branch?

**Comment by tteschemacher** [comment]
nope, is gone, thanks!

**Pull request merged**